Userland/Libraries/LibWeb/Animations/AnimationEffect.h

```diff
@@ -148,6 +148,11 @@ class AnimationEffect : public Bindings::PlatformObject {
     virtual DOM::Element* target() const { return {}; }
     virtual bool is_keyframe_effect() const { return false; }
 
+    virtual void invalidate()
+    {
+        VERIFY_NOT_REACHED();
```

Comment:
If the goal is to ensure subclasses always implement this, then leaving it a pure virtual would be better.

```c++
    virtual void invalidate() = 0;
```